Runtime/Scripts/WebRTC.cs

```diff
@@ -48,18 +48,34 @@ public RTCDataChannelInit(bool reliable)
     public class RTCRtpReceiver
     {
         internal IntPtr self;
-        internal RTCRtpReceiver(IntPtr ptr)
+        private RTCPeerConnection peer;
+
+        internal RTCRtpReceiver(IntPtr ptr, RTCPeerConnection peer)
         {
             self = ptr;
+            this.peer = peer;
+        }
+
+        public RTCStatsReportAsyncOperation GetStats()
+        {
+            return peer.GetStats(this);
         }
     }
 
     public class RTCRtpSender
     {
         internal IntPtr self;
-        internal RTCRtpSender(IntPtr ptr)
+        private RTCPeerConnection peer;
```

Comment:
This class seems to be the same as `RTCRtpReceiver.`
Maybe we can combine them ?

Reply:
This pull request has the changes for `RTCRtpSender` class.
It could be better as it is.
https://github.com/Unity-Technologies/com.unity.webrtc/pull/118/files#diff-7ef9c5b7b61c1912384f936f85cc13dcR60-R94